xla/service/stream_pool.cc

```diff
@@ -58,7 +68,9 @@ void StreamPool::ReturnStream(se::Stream* stream) {
     VLOG(1) << stream->DebugStreamPointers()
             << " StreamPool returning ok stream";
     absl::MutexLock lock(&mu_);
-    streams_.emplace_back(stream);
+    stream_executor::StreamPriority priority =
```

Comment:
ReturnStream doesnt have an input priority, I didnt change the interface of this function since we can just extract the priority from the stream itself.